platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/ConfigurationAdminMigratorTest.java

```diff
@@ -41,18 +44,26 @@
 import java.util.Iterator;
 import java.util.Map;
 
+import org.apache.commons.io.FileUtils;
 import org.codice.ddf.configuration.status.ConfigurationStatus;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
 import org.osgi.framework.InvalidSyntaxException;
 import org.osgi.service.cm.Configuration;
 import org.osgi.service.cm.ConfigurationAdmin;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
-@RunWith(MockitoJUnitRunner.class)
-public class ConfigurationFileDirectoryTest {
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ConfigurationAdminMigrator.class, FileUtils.class})
+public class ConfigurationAdminMigratorTest {
```

Comment:
This unit test class is _way_ too big and complex. I started working on cleaning it up/simplifying it. Hopefully I'll be done before code freeze.